### PR TITLE
Instructor: home page: remove extra space at end of course panels #7502

### LINKED ISSUE #7502: Instructor: home page: remove extra space at end of course panels
At the end of each course panel, there's an unwanted extra space. This needs to be removed.

capture

### PR DESCRIPTION
**Pull request by amarlearning**
Fixes #7502 

_[Screenshot]_
screenshot from 2017-06-12 02-04-18

### CODE REVIEW

**Comment by wkurniawan07** [comment]
While this patch works, do you happen to know what caused the extraneous space in the first place? Sometimes, tackling the root cause is better than simply remedying the symptoms.

**Comment by amarlearning** [comment]
@wkurniawan07 Bootstrap table by default has a `margin: 15px`. That's why we had this issue.

**Comment by damithc** [comment]
> @wkurniawan07 Bootstrap table by default has a margin-bottom: 20px. That's why we had this issue.

What made it appear all of a sudden though? We didn't have it before, so it must be something we did in the recent few PRs.

**Comment by amarlearning** [comment]
> What made it appear all of a sudden though? We didn't have it before, so it must be something we did in the recent few PRs.

Instead of proving an external class to give margin to any table, someone might have altered the default global classes like `table`, `table-responsive`, `table-striped`, `table-bordered`. Due to which all the table that are present in teammates might be affected.

**Comment by VamsiSangam** [comment]
It was unintentionally introduced in #7389 . [Diff](https://github.com/TEAMMATES/teammates/pull/7389/files#diff-6a74162f9bab98a8199093a17b80a49a).

To create a collapsible panel, you need to wrap it in `<div class="panel-collapse collapse">`, which is what is done in the PR.

**Earlier**

Earlier, the `.table-responsive` was a direct child of `panel`, due to which some special styles get added, which overrides the default `margin-botton:20px` of `table`.



**Now**

Now the `.table-responsive` is not a direct child of `panel`, so that style which applied earlier is not applicable now. So now the default `margin-botton:20px` of `table` is not overriden, which is why we get that extra space.



@amarlearning 's solution is perfect. :+1: And BTW @amarlearning you can't change the bootstrap code because we don't store those CSS files, it is loaded via CDN :smile:

**Comment by amarlearning** [comment]
@VamsiSangam Not talking about changing code inside bootstrap's file. Consider this example:

File name: `Bootstrap.min.css`
```css
.table {
    width: 100%;
    margin-bottom: 10px;
}
```

File name: `teammatesCommon.css`
```css
.table {
    margin-bottom: 20px;
}
```
Now, in this case, I didn't modified any code inside `bootstrap.min.css` but instead, I modified a class that is defined in `bootstrap.min.css` inside `teammatesCommon.css`

**Comment by VamsiSangam** [comment]
Ok! Sorry for misreading :sweat_smile:

**Comment by whipermr5** [comment]
@amarlearning Pending GodMode run for two tests :)

File: src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html [hunk 1]
```diff
@@ -213,7 +213,7 @@ <h5 class="col-sm-2 margin-top-0">
                 <div class="col-sm-10">
                   <div class="panel panel-default panel-body mce-content-body content-editor" id="instructions" spellcheck="false">
                     <p>
-                      updated instructions
+                      Please give your feedback based on the following questions.
```
Comment (whipermr5) [inline src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html]: This is causing the build failure; revert it and it should pass.

Comment (amarlearning) [inline src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html]: I don't understand why everytime this line cause builds failure.
If all the CI test got passed in the Godmode then how come this build is failing :thinking:

Comment (whipermr5) [inline src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html]: You need to re-run the test again with GodMode switched off to ensure it works. More details why this is so can be found in the [GodMode doc](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md#final-notes).

**Pull request merged**